src/command/upload.ts

```diff
@@ -320,4 +319,32 @@ export class Upload extends RootCommand implements LeafCommand {
 
     return false
   }
+
+  private async handleSyncSupport(): Promise<void | never> {
+    if (this.quiet || !this.sync) {
```

Comment:
This makes it not possible to turn on syncing in quiet mode?

Reply:
This is only about printing the warnings. Sync will work regardless of verbosity level.

Reply:
Then maybe it would be better to rename the function to `printWarningWhenNoConnectedPeers` or something more relevant.